### ISSUE TITLE
Modularize Backend: Low hanging fruit

### ISSUE THREAD

**Issue by kayline** [department-of-veterans-affairs/va.gov-team#97410]
## Value Statement

**_As a_** vets-api developer

**_I want to_** see how much of our code I can move into an engine quickly

**_So that_** so I can get a sense for whether the effort is worth it

---

## Background Context

- An initial spike for #97045 

## Outcome, Success Measure, KPI(S), and Tracking Link

- Spend 3 days, see what we can get isolated

## Acceptance Criteria

- [ ] _AC_
- [ ] _AC_
- [ ] _AC_

## Designs and Build Notes

- Add here

### Enablement team (if needed)
@ member of team

## Out of scope

- Add here

## Open questions

- Add here

## Tasks

- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by eileen-coforma** [comment]
Started yesterday.

**Comment by eileen-coforma** [comment]
Draft PR is open, engine is created.

TLDR: Attempt to move all of the code we own into a contained engine. Spike has moved sidekiq jobs and controls. ENG team needs to have convo to talk about jobs and controls, and whether we need models moved as well. Doable 80% easy 20% will be difficult.

**Comment by eileen-coforma** [comment]
Continued work in progress. Pulling more pieces into the contained. Need to meet with ENG to make progress.

**Comment by Mottie** [comment]
This investigation is done, will follow up with more tickets on implementation.
